config/terraform/aws/s3.tf

```diff
@@ -44,6 +60,27 @@ resource "aws_s3_bucket" "cloudfront_logs" {
   #tfsec:ignore:AWS002
 }
 
+resource "aws_s3_bucket" "exposure_config_logs" {
+  bucket = "covid-shield-exposure-config-${var.environment}-logs"
+  acl    = "log-delivery-write"
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm = "AES256"
+      }
+    }
+  }
+
+  lifecycle_rule {
+    enabled = true
+
+    expiration {
+      days = 90
+    }
+  }
+  #tfsec:ignore:AWS002
```

Comment:
Can we have a comment here on what this rule is and why it's being ignored, instead of requiring a dev look it up in the TFSec project.

Reply:
Added comments